RFC-0004-jdbc-join-pushdown.md

```diff
@@ -0,0 +1,429 @@
+# **RFC-0004-jdbc-join-pushdown for Presto**
+
+See [CONTRIBUTING.md](CONTRIBUTING.md) for instructions on creating your RFC and the process surrounding it.
+
+## Jdbc join pushdown in presto
+
+Proposers
+
+* Ajas M M
+
+
+## [Related Issues]
+
+https://github.com/prestodb/presto/issues/23152
+
+## Summary
+
+When presto identifies a join query which is specific to a Jdbc remote datasource, it split the join query into multiple select query based on the tables involved in the join query and select all the records from each tables using the jdbc connector without passing the join condition. Then the results of these sub-queries are fetched into the presto workers, additional operations such as filters, joins, sorts are applied before sending back to the user.
```

Comment:
Please can you add an example here to demonstrate clearly the JDBC pushdown and improvement.

Show a query, original plan, new plan with JDBC pushdown and the improvements.

Reply:
Added details like query, original plan, joinpushdown plan and performance here